enterprise/server/cmd/ci_runner/main.go

```diff
@@ -656,6 +660,27 @@ func run() error {
 	return nil
 }
 
+// parseFlags should not fail when parsing an undefined flag.
+// This lets us add new flags to this script without breaking older executors
+// (on self-hosted executors, for example) that aren't expecting them when the
+// app server tries to send them.
+func parseFlags() error {
+	// Silence noisy output when parsing an unknown flag
```

Comment:
nit: this would let us avoid having to use flagset above

```suggestion
	// Ignore errors when parsing an unknown flag
	flagset := flag.CommandLine
	flagset.Init(os.Args[0], flag.ContinueOnError)
```

Reply:
Done